src/AppBundle/Entity/Site.php

```diff
@@ -112,6 +131,42 @@ class Site implements AuthorEditorable, \JsonSerializable
     private $framework;
 
     /**
+     * @var int
+     *
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User", inversedBy="sitesDevelopedBy")
+     * @ORM\JoinColumn(onDelete="SET NULL", nullable=true)
+     */
+    private $developer;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $developerName;
+
+    /**
+     * @var int
+     *
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User", inversedBy="sitesManagedBy")
+     * @ORM\JoinColumn(onDelete="SET NULL", nullable=true)
+     */
+    private $responsibleManager;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $managerName;
+
+    /**
+     * @var \DateTime
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    protected $endDate;
```

Comment:
Please find out and make it clear, my guess is end of SLA, so it should be called `slaEndAt`

Its also probably a `date` type not `datetime`